python/shotgun_desktop/initialization/install.py

```diff
@@ -190,13 +195,16 @@ def create_structure(self):
             fh.write(self._executables[x])
             fh.close()
 
+    def _get_app_store_proxy(self):
+        return self._app_store_http_proxy or self._sg_proxy
```

Comment:
I thought about that case, but since I couldn't think of a good reason why someone would configure it this way I haven't bothered. It would need to be implemented in core too, not just in the startup. We can always implement something if this ever becomes a requirement for a client.